php/post-types/class-block-post.php

```diff
@@ -205,26 +205,13 @@ public function register_post_type() {
 			'show_ui'       => true,
 			'show_in_menu'  => true,
 			'menu_position' => 100,
-			// This SVG icon is being included from the plugin directory, so using file_get_contents is okay.
-			// @codingStandardsIgnoreStart
-			'menu_icon'     => 'data:image/svg+xml;base64,' . base64_encode(
-				file_get_contents(
-					$this->plugin->get_assets_path( 'images/admin-menu-icon.svg' )
-				)
+			'menu_icon'     => 'data:image/svg+xml;base64,' . base64_encode( // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_encode
+				file_get_contents( $this->plugin->get_assets_path( 'images/admin-menu-icon.svg' ) ) // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents -- This SVG icon is being included from the plugin directory, so using file_get_contents is okay.
 			),
-			// @codingStandardsIgnoreEnd
 			'query_var'     => true,
 			'rewrite'       => array( 'slug' => $this->slug ),
 			'hierarchical'  => true,
-			'capabilities'  => array(
-				'edit_post'          => 'block_lab_edit_block',
-				'edit_posts'         => 'block_lab_edit_blocks',
-				'edit_others_posts'  => 'block_lab_edit_others_blocks',
-				'publish_posts'      => 'block_lab_publish_blocks',
-				'read_post'          => 'block_lab_read_block',
-				'read_private_posts' => 'block_lab_read_private_blocks',
-				'delete_post'        => 'block_lab_delete_block',
-			),
+			'capabilities'  => $this->get_capabilities(),
```

Comment:
There should be no change here, `$this->get_capabilities()` simply wraps the previous code.